.github/workflows/test_ollama.yml

```
@@ -0,0 +1,70 @@
+          done
+
+      - name: Pull Model
+        run: |
+          curl -X POST http://localhost:11434/api/pull -d '{"name": "ollama/llama3.2"}'
+      - name: Pull Embedding Model
+        run: |
+          curl -X POST http://localhost:11434/api/pull -d '{"name": "avr/sfr-embedding-mistral:latest"}'
+          
+
+  run_simple_example_test:
+      uses: ./.github/workflows/reusable_python_example.yml
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)